src/Extensions/voice_control.py

```diff
@@ -0,0 +1,137 @@
+import discord
+from discord.ext import commands
+from src.util import VOICE_PERMITTED_ROLES_NAMES, have_permitted_rol, str_permitted_roles_names
+
+
+def setup(bot):
+    bot.add_cog(VoiceControl(bot))
+
+
+class VoiceControl(commands.Cog):
+
+    def __init__(self, bot_):
+        self.bot = bot_
+        self.levantados_ = []
+
+    @commands.command(name='Levantados', aliases=['levantados'],
+                      brief='Muestra los usuarios que han levantado la mano',
+                      description='Muestra, en orden de llegada, los usuarios que han levantado la mano')
+    async def levantados(self, ctx):
+        message = ctx.message
+        message_channel = message.channel
+        if len(self.levantados_) == 0:
+            await message_channel.send('Nadie ha levantado la mano')
+        else:
+            result = 'Los usuarios que han levantado la mano, en orden de llegada ' \
+                     'siendo el de mas a la izquierda el primero, son:\n' + ", ".join(self.levantados_)
+            await message_channel.send(result)
+
+    @commands.command(name='LevantarMano', aliases=['levantarmano', 'Levantar', 'levantar'],
+                      brief='Levanta la mano(solicita desilencio)',
+                      description='Permite levantar la mano para que, posteriormente, un ' +
+                                  str_permitted_roles_names(VOICE_PERMITTED_ROLES_NAMES) + ' te de la palabra')
```

Comment:
En vez de sumar strings, Python tiene alguna opcion para construir strings que es mas comoda de leer.
```suggestion
                      description='Permite levantar la mano para que, posteriormente, un {} te de la palabra'.format(str_permitted_roles_names(VOICE_PERMITTED_ROLES_NAMES))
```

Reply:
no crees que es mejor usar f"String" en vez de "String".format

Reply:
Si, f"string" parece ser la mejor forma, la verdad que la desconocia y la he tenido que buscar pero me gusta. 

Además parece ser la mas eficiente y en cuanto a sintaxis la mas clara.